e2e/testcases/override_reconcile_timeout_test.go

```diff
@@ -85,7 +99,7 @@ func TestOverrideReconcileTimeout(t *testing.T) {
 	} else {
 		expectActuationStatus := "Succeeded"
 		expectReconcileStatus := "Timeout"
```

Comment:
This expectation will pass today, but after the RGC bug is fixed, the reconcile status may no still be Timeout. Inc fact, the longer you wait, the less likely it will still be Timeout. So this might be a race condition.

Reply:
What wait time are you discussing here?  The reconcile status is updated as timeout by the Config Sync reconciler at the end of sync operation. WaitforRepoSyncs() only waits until the sync operation finishes, and we check the reconcile status right after it. After the RGC bug is fixed, the resourcegroup controller will also update the reconcile status. For this "timeout" case, when the resourcegroup controller changes the reconcile status from timeout to current, the reconcile status check is already finished in the test.

Reply:
Ok, I think if the RGC preserves the Timeout status instead of reverting to InProgress/Pending, we should be ok for up to 5s. If it's flakey afterwards tho, we may need to fix it later.